### PR TITLE
Fix/152

### PR DESCRIPTION
**Pull request by Muhaddatha**
Markdown images are visible
Spacing under preview box is adjusted
Link underline appears only when hovered over

### CODE REVIEW

File: applications/osb-portal/src/pages/RepositoryPage.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ const useStyles = makeStyles((theme) => ({
       paddingRight: 0,
       marginTop: theme.spacing(2),
       [theme.breakpoints.up("sm")]: {
-        maxHeight: "calc(100vh - 19.5rem)",
+        maxHeight: "calc(100vh - 15.0rem)",
```
Comment (filippomc) [inline applications/osb-portal/src/pages/RepositoryPage.tsx:107]: this is not the better way to fit the vertical space. Better use flex with stretch

File: applications/osb-portal/src/pages/RepositoryPage.tsx [hunk 2]
```diff
@@ -250,17 +250,22 @@ const useStyles = makeStyles((theme) => ({
...
+        marginTop: theme.spacing(2),
         backgroundColor: 'rgba(0, 0, 0, 0.25)',
         boxShadow: '0px 0px 0px 3px rgba(0, 0, 0, 0.25)',
         borderRadius: radius,
         fontFamily: font,
         overflow: "auto",
         flexGrow: 1,
         [theme.breakpoints.up("sm")]: {
-          height: "calc(100vh - 14.5rem)",
+          height: "calc(100vh - 15.0rem)",
```
Comment (filippomc) [inline applications/osb-portal/src/pages/RepositoryPage.tsx:261]: see above

File: applications/osb-portal/src/pages/RepositoryPage.tsx [hunk 3]
```diff
@@ -433,6 +437,33 @@ export const RepositoryPage = () => {
...
+    }
+    return [imageTags, imagePaths];
+  }
+
+  const convertImgInMarkdown = (markDown: string, repositoryUri: string) => {
+    let mark = markDown;
+    const [imageTags, imagePaths] = getImages(markDown);
+    const updatedImages : string[] = [];
+    imagePaths.map((image) => {
+      const y = `[![img](${repositoryUri.replace('https://github.com/', 'https://raw.githubusercontent.com/')}/master/${image})](${repositoryUri.replace('https://github.com/', 'https://raw.githubusercontent.com/')}/master/${image})?target=_blank`;
```
Comment (filippomc) [inline applications/osb-portal/src/pages/RepositoryPage.tsx:458]: must take repository uri and context ( branch may be different from master). Also, images may come from outside the repo (can be any in a markdown). Need to think about this

**Review by filippomc — CHANGES_REQUESTED** [review]
Need to think more about vertical space filling and how to get images path.

File: applications/osb-portal/src/pages/RepositoryPage.tsx [hunk 4]
```diff
@@ -103,19 +108,18 @@ const useStyles = makeStyles((theme) => ({
...
-      height: "calc(100vh)",
+      height: '100%',
     },
     "& .main-content": {
       padding: theme.spacing(3),
-      overflow: "auto",
-
-      [theme.breakpoints.up("sm")]: {
-        height: "calc(100%)",
+      height: '83vh',
```
Comment (filippomc) [inline applications/osb-portal/src/pages/RepositoryPage.tsx:115]: Still not ideal, not understanding why we are setting vertical height, I may be missing something

Comment (Muhaddatha) [reply applications/osb-portal/src/pages/RepositoryPage.tsx:115]: I don't know how to fix the heights.

**Review by filippomc — APPROVED** [review]
good to merge

**Pull request merged**